pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -271,7 +271,7 @@ func (alloc *Action) allocateResourcesForTasks(tasks *util.PriorityQueue, job *a
 				metrics.UpdateE2eSchedulingLastTimeByJob(job.Name, string(job.Queue), job.Namespace, time.Now())
 			}
 		} else {
-			klog.V(3).Infof("Predicates failed in allocate for task <%s/%s> on node <%s> with limited resources",
```

Comment:
Is this a normal case here? because when cluster has no idle resources, predict will be failed and allocate for task will be breaked, and will not go through here.
But it's a reasonable modification, just want to know why codes go here: )